scripts/valhalla_build_config

```diff
@@ -579,7 +582,12 @@ def add_leaf_args(path, tree, leaves, parser):
       h = h[k]
     t = comma_separated_list if type(tree) == type([]) else (type(tree) if type(tree) != type(optional()) else tree.type)
     arg = '--' + path.replace('_', '-').replace('\0', '-')
-    parser.add_argument(arg, type = t if t != type(bool()) else parse_bool, help=h, default=tree)
+    if t == type(bool()):
+      t = parse_bool
+    elif t == type(list()):
+      t = parse_list
```

Comment:
if you wanna get rid of a few lines (untested), you can inline those functions:

```suggestion
    if t == type(bool()):
      t = lambda arg: bool(strtobool(arg))  # from distutils.util import strtobool
    elif t == type(list()):
      t = lambda arg: eval(arg)
```

Reply:
i like this suggestion, let me make the changes